ssd1306/image1bit/image1bit.go

```diff
@@ -143,6 +143,20 @@ func (i *VerticalLSB) SetBit(x, y int, b Bit) {
 	}
 }
 
+// Draw a horizontal line from start to end at ypos
+func (i *VerticalLSB)DrawHLine(start, end, ypos int, b Bit) {
```

Comment:
You may want to configure your editor to format on save. This is quite useful as then you can type without caring about formatting and a quick file save formats for you.